exercises/practice/bowling/.meta/gen.go

```diff
@@ -1,81 +1,62 @@
 package main
 
 import (
+	"../../../../gen"
```

Comment:
The import order was correct in the old version.

In Go, it is idiomatic to have 3 groups of import statements separated by an empty line.
1. standard library
2. external dependencies
3. local packages

Some people switch or combine 2 and 3 but in general stdlib imports always come first and are a group of its own.
If your IDE gets this wrong you might need to adjust the settings. Some tools like goimports have trouble getting the groups right automatically currently (there are open issues about this) but they will usually honor the groups if you set them up manually.

Reply:
I have adjusted the settings, I think it is correct now